rust/private/rust.bzl

```diff
@@ -602,6 +602,11 @@ _common_attrs = {
     "_cc_toolchain": attr.label(
         default = "@bazel_tools//tools/cpp:current_cc_toolchain",
     ),
+    "_collect_cc_coverage": attr.label(
+        default = "//util:collect_coverage",
+        executable = True,
+        cfg = "exec",
```

Comment:
This seems right to me, but upstream they use the target cfg 🤔 https://github.com/bazelbuild/bazel/blob/45f462893df4d55cee19c05ab01e051e229597a3/src/main/starlark/builtins_bzl/common/cc/semantics.bzl#L100